src/Libraries/DesignScriptBuiltin/DesignScriptBuiltin.csproj

```diff
@@ -11,10 +11,13 @@
     <DocumentationFile>$(OutputPath)\$(UICulture)\DesignScriptBuiltin.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" >
-      <ExcludeAssets>runtime</ExcludeAssets>
+    <PackageReference Include="System.Collections.Immutable" Version="5.0.0">
       <GeneratePathProperty>true</GeneratePathProperty>
+      <IncludeAssets>none</IncludeAssets>
     </PackageReference>
+    <Reference Include="System.Collections.Immutable">
+      <HintPath>$(PkgSystem_Collections_Immutable)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
+    </Reference>
```

Comment:
By including `none` and adding an assembly reference to the immutable dll we are forcing `DesignScriptBuiltin.csproj` to reference the `portable-net45...` dll...and not whatever targetframework specific dll it automatically found before..